app/src/main/java/io/github/pietrocaselani/moviestraker/interactors/GenresInteractorInput.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.pietrocaselani.moviestraker.interactors
+
+import io.github.pietrocaselani.moviestraker.tmdb.entities.GenreListResponse
+import io.reactivex.Flowable
+
+/**
+ * Created by pc on 25/06/17.
+ */
+interface GenresInteractorInput {
```

Comment:
I'm used to develop iOS apps using VIPER architecture, and I like the concept of "interactors" on VIPER, and I wanted a compoenent responsible for interacting with the API or database.
ViewModels depends on interfaces so it can be easly modified (they don't need to know how or where the data comes) or tested, using mocks or something like that.